templates/react/field-types/Vec/edit/render.hbs

```diff
@@ -0,0 +1,7 @@
+<h4>{{title_case field_name}}</h4>
+{#each {{camel_case field_name}} as el, i}
+  {{> (concat field_type.type "/" widget "/edit/render") label="" variable_to_read="el" variable_to_change=(concat (camel_case field_name) "[i]" ) }} }`)}
+{/each}
+<button onClick={() => set{{pascal_case field_name}}([...{{camel_case field_name}}, {{> (concat field_type.type "/" widget "/initial-value") field_type=field_type}}])}>
```

Comment:
This doesn't seem to add another element to the list in the new-entry form -- at least it doesn't result in a new widget in the UI. And as soon as I click it, the 'Create <entry type>' or 'Edit <entry type>' button is disabled.

Reply:
Fixed